setup.py

```diff
@@ -78,7 +78,7 @@
         "Topic :: Scientific/Engineering",
     ],
     keywords="qiskit sdk quantum",
-    python_requires=">=3.7",
+    python_requires=">=3.9",
```

Comment:
Python 3.7 is already EOL and 3.8 will be at the end of this month.

Reply:
makes sense, ty again!